bitnami/kafka/README.md

```diff
@@ -109,6 +109,8 @@ The following tables lists the configurable parameters of the Kafka chart and th
 | `extraVolumeMounts`                               | Extra volumeMount(s) to add to Kafka containers                                                                                   | `[]`                                                    |
 | `auth.clientProtocol`                             | Authentication protocol for communications with clients. Allowed protocols: `plaintext`, `tls`, `mtls`, `sasl` and `sasl_tls`     | `plaintext`                                             |
 | `auth.interBrokerProtocol`                        | Authentication protocol for inter-broker communications. Allowed protocols: `plaintext`, `tls`, `mtls`, `sasl` and `sasl_tls`     | `plaintext`                                             |
+| `auth.saslMechanisms`                             | SASL mechanisms when either `auth.interBrokerProtocol` or `auth.clientProtocol` are `sasl`. Allowed types: `plain`, `scram-sha-256`, `scram-sha-512` | `nil`                                |
```

Comment:
are `sasl` or `sasl_tls`

Reply:
Shouldn't the default value be set to every allowed type (to be compatible with previous version)?

Reply:
Yes, actually I set it on the image, I have to update it here too.